cogs/audio.py

```diff
@@ -763,13 +763,16 @@ def _match_any_url(self, url):
         while d.is_alive():
             await asyncio.sleep(0.5)
 
-        for entry in d.song.entries:
-            if entry["url"][4] != "s":
-                song_url = "https{}".format(entry["url"][4:])
-                playlist.append(song_url)
-            else:
-                playlist.append(entry.url)
-
+        try:
+            for entry in d.song.entries:
+                if entry["url"][4] != "s":
+                    song_url = "https{}".format(entry["url"][4:])
+                    playlist.append(song_url)
+                else:
+                    playlist.append(entry.url)
+        except:
```

Comment:
Don't have catch all exception handlers here. Also try to minimize the amount of code inside of a `try/except` and explicitly catch exceptions.